utils/helpers.js

```diff
@@ -181,3 +181,34 @@ export const getSample = (field) => {
     throw new Error(`Invalid model type:${field.type}`);
   }
 }
+
+export const calculateRefundTotal = (booking, policy) => {
+  const { price: { val: initialValue }, start, volunteer } = booking
+  if (volunteer) return 0
```

Comment:
@closerearth here is the change I was mentioning in the chat, if a guest booked his stay as a volunteer we just return 0, let me know if that will work

Reply:
That's fine for now yes